testing-harness/test/execution.test.js

```diff
@@ -80,7 +80,7 @@ test('Should only load elm JSON with the specified identifier', () => {
 
 test('Should default to loading elm with the mCODE identifier', () => {
   // Pulling elm with the mCODE identifier along with its valueSetMap
-  const valueSets = loadValueSets();
+  const valueSets = loadValueSets('./test/fixtures/valuesets');
```

Comment:
Is this because this is relative to where the `loadValueSets` function itself is? If so, maybe it makes sense to coordinate this with #14 as well, but it might be better to do the `path.join()` outside of the function so that the function is receiving an absolute path.